script/vsts/platforms/templates/get-release-version.yml

```diff
@@ -0,0 +1,17 @@
+jobs:
+
+- job: GetReleaseVersion
+  pool:
+    vmImage: 'ubuntu-latest'
+  steps:
+    # This has to be done separately because VSTS inexplicably
+    # exits the script block after `npm ci` completes.
+    - script: |
+        cd script/vsts
+        npm ci
+      displayName: npm ci
+    - script: node script/vsts/get-release-version.js
```

Comment:
This does not include `--nightly` for the nightly build

Reply:
> This does not include `--nightly` for the nightly build

Thanks for noticing this. It does now!